pkg/config/filter_test.go

```
@@ -0,0 +1,715 @@
+			if (err != nil) != tt.wantErr {
+				t.Errorf("Evaluate() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if !tt.wantErr && got != tt.want {
+				t.Errorf("Evaluate() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestFilterNodeValidate(t *testing.T) { //nolint:maintidx // table-driven test with many cases
```

golangci-lint (2.11.4)

[error] directive `//nolint:maintidx // table-driven test with many cases` is unused for linter "maintidx"

(nolintlint)